docs/guide/url_management.md

```diff
@@ -120,3 +120,34 @@ app.url_checker.handler.custom:
 ```
 
 The `scheme` attribute is mandatory and has to correspond to the name of the protocol, for instance `ftp`.
+
+## URL alias patterns
+
+You can configure how eZ Platform generates URL aliases. The configuration is available under `ezpublish.url_alias.slug_converter`, for example:
+
+``` yaml
+ezpublish:
+    url_alias:
+        slug_converter:
+            transformation: example_group
+            separator: dash
+            transformation_groups:
+                example_group:
+                    commands:
+                        - 'space_normalize'
+                        - 'hyphen_normalize'
+                        - 'apostrophe_normalize'
+                        - 'doublequote_normalize'
+                        - 'your_custom_command'
+                    cleanup_method: url_cleanup
+```
+
+`transformation` indicates which pattern will be used by default.
+The `transformation_groups` key contains the available patterns for URL generation.
+There are three types of `separator` available: `dash`, `underscore` and `space`.
+
+A transformation group consists of an array of commands (see [all available commands](https://github.com/ezsystems/ezpublish-kernel/tree/master/eZ/Publish/Core/Persistence/Tests/TransformationProcessor/_fixtures/transformations)) and a [`cleanupMethod`](https://github.com/ezsystems/ezpublish-kernel-ee/blob/master/eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/SlugConverter.php#L290).
```

Comment:
Would it be better to list and explain all of the commands in the doc ?

Reply:
But wouldn't that be basically copying the comments from those files verbatim into the doc? IMHO they are pretty clear, like here: https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/Persistence/Tests/TransformationProcessor/_fixtures/transformations/basic.tr#L22

Reply:
Maybe it is sufficient. I'm not a big fan of linking to things in code that way (for pure doc purpose, as opposed to examples), as there are real chances it changes at some point. Maybe link to the current kernel stable branch at least (instead of master) ?

Reply:
Fair enough, I'll improve this with a summary of the commands and a link to a stable branch